node/src/main/scala/coop/rchain/node/ReplClient.scala

```diff
@@ -28,15 +25,20 @@ class GrpcReplService(host: String, port: Int) extends ReplService[Task] {
   def run(line: String): Task[String] = Task.delay {
     blockingStub.run(CmdRequest(line)).output
   }
+  def eval(fileNames: List[String]): Task[String] =
```

Comment:
Is this being used anywhere? I can't see it.

Reply:
Yes. It is called via the ReplService trait in node/.../repl.scala:39